tests/slurm_plugin/test_clustermgtd.py

```diff
@@ -1840,7 +1846,7 @@ def test_manage_cluster_boto3(
     # patch boto3 call
     boto3_stubber("ec2", mocked_boto3_request)
     mocker.patch("slurm_plugin.clustermgtd.datetime").now.return_value = datetime(2020, 1, 2, 0, 0, 0)
-    mocker.patch("slurm_plugin.clustermgtd.read_json", side_effect=[FLEET_CONFIG, LAUNCH_OVERRIDES])
+    mocker.patch("slurm_plugin.clustermgtd.read_json", side_effect=[FLEET_CONFIG, LAUNCH_OVERRIDES, LAUNCH_OVERRIDES])
```

Comment:
Since we'll be having two `overrides` files, can we make the tests explicitly check that the correct overrides type is being applied. At the moment `LAUNCH_OVERRIDES` is an empty JSON object`{}` so we can't be 100% sure if the correct content is being read.

It would be great to have a `RUN_INSTANCE_LAUNCH_OVERRIDES` and `CREATE_FLEET_LAUNCH_OVERRIDES` with corresponding test content for proper comparison.

Reply:
I left an empty json object for the other test cases on which we're not really using these parameters.

I have this kind of tests, with different content for the two files, in the `TestCreateFleetManager.test_evaluate_launch_params` and `TestRunInstancesManager.test_evaluate_launch_params`, in the `test_fleet_manager.py` module. 

Could you double check if this is what you're searching for?

Reply:
Thanks for referencing those tests.
- [TestEc2RunInstancesManager](https://github.com/aws/aws-parallelcluster-node/blob/cd6781f2f5fc167052a61151d4c9611e67a5ef33/tests/slurm_plugin/test_fleet_manager.py#L130-L130)
- [TestCreateFleetManager](https://github.com/aws/aws-parallelcluster-node/blob/cd6781f2f5fc167052a61151d4c9611e67a5ef33/tests/slurm_plugin/test_fleet_manager.py#L326-L326)